src/app/components/music-controls/music-controls.component.html

```diff
@@ -0,0 +1,43 @@
+<div class="controls-wrapper">
+    <div class="play-controls">
+        <button class="transparent_button" (click)="playPreviousTrack()">
+            <i class="material-icons" id="arrow">arrow_left</i>
+        </button>
+        <button class="transparent_button" (click)="playNextTrack()" [disabled]="isTracksListEnd">
+            <i class="material-icons" id="arrow" [style.opacity]="isTracksListEnd ? '0.6' : '1'">arrow_right</i>
+        </button>
+    </div>
+    <button class="transparent_button" (click)="autorenew()">
+        <i class="material-icons" [style.opacity]="isAutorenewed ? '1' : '0.6'">autorenew</i>
+    </button>
+    <button class="transparent_button" (click)="playPause()">
+        <i class="material-icons" id="play" *ngIf="state?.playing">pause_circle_outline</i>
+        <i class="material-icons" id="pause" *ngIf="!state?.playing">play_circle_outline</i>
+    </button>
+    <div class="audio-details">
+        <div class="track-details">
+            <div class="audio-details">
+                <h5>{{currentTrack?.name}}</h5>
+                <h6>{{currentTrack?.artists[0].name}}</h6>
+            </div>
+            <p>{{state?.readableCurrentTime}}</p>
+        </div>
+        <mat-slider class="time-slider" min="0" [max]="state?.duration" step="1" [value]="state?.currentTime"
+            (input)="rewindTo($event)">
+        </mat-slider>
+    </div>
+    <button class="transparent_button" (click)="shuffle()">
+        <i class="material-icons" [style.opacity]="isShuffled ? '1' : '0.6'">shuffle</i>
+    </button>
+    <div class="play-controls">
+        <button class="transparent_button" *ngIf="!isVolumeOff">
+            <i class="material-icons">volume_up</i>
```

Comment:
The same as above ({{}} instead of *ngIf). + You're using button element without any action - use `div` instead